### PR TITLE
fix sorts, resolves #5979

### LINKED ISSUE #5979: Sorting by label
In 
  http://www.lmfdb.org/padicField/?n=8&p=2&c=18&hidecol=p.c.e.f.gal.slopes
the fields are helpfully sorted by label, but instead of 
  2.8.18.1, 2.8.18.2, ... 
it does
  2.8.18.1, 2.8.18.10, ...
(sorting the strings).  Note that sorting by label is not an option in the dropdown list--perhaps we could add that, since we using it to break ties.  But wouldn't it also make more sense to sort them in the first way?  (It would make my life easier, when I'm grabbing number 18 from the list, it's easier when that's in the 18th place.)  

I have a workaround, so this is low priority.

### PR DESCRIPTION
**Pull request by AndrewVSutherland**

### CODE REVIEW

File: lmfdb/modular_curves/main.py [hunk 1]
```diff
@@ -962,11 +962,12 @@ def __init__(self):
             [CPlabel],
         ]
 
+    _default = ["level", "index", "genus", "coarse_class_num", "coarse_level", "coarse_num", "fine_num"]
     sorts = [
-        ("", "level", ["level", "index", "genus", "label"]),
-        ("index", "index", ["index", "level", "genus", "label"]),
-        ("genus", "genus", ["genus", "level", "index", "label"]),
-        ("rank", "rank", ["rank", "genus", "level", "index", "label"]),
+        ("level", "level", _default),
```
Comment (edgarcosta) [inline lmfdb/modular_curves/main.py:967]: I think we want `"", "level", _default` for the first entry

Comment (AndrewVSutherland) [reply lmfdb/modular_curves/main.py]: Done (although I don't think it actually matters).

Comment (roed314) [reply lmfdb/modular_curves/main.py]: The way it matters is just in the url: if you have a blank default sort option it doesn't clutter urls.

**Comment by roed314** [comment]
For the rank sort, do we want genus to be next (as was done before), or just `["rank"] + _default`?

**Comment by edgarcosta** [comment]
lgtm

**Comment by edgarcosta** [comment]
> For the rank sort, do we want genus to be next (as was done before), or just `["rank"] + _default`?

genus next

**Pull request merged**